n2y/blocks.py

```diff
@@ -469,9 +469,21 @@ class LinkPreviewBlock(WarningBlock):
 
 
 class SyncedBlock(Block):
+    def __init__(self, client, notion_data, page, get_children=True):
+        super().__init__(client, notion_data, page, get_children)
```

Comment:
Will the `Block.__init__` also call `self.client.get_child_blocks`? If so, since we're over-riding the `self.children` property here, maybe we could pass in `get_children=False` to avoid an unnecessary API call?

Reply:
Hm, good question.

It's only overriding self.children when `not self.original and self.shared`. So the only time it would be performing an unnecessary API call might be when it's a reference synced block on the same page. I can pass the `self.original` flag into the super constructor and that should remove all redundant API calls